### ISSUE TITLE
Add Swagger

### ISSUE THREAD

**Issue by KaiVolland** [terrestris/gondolin#12]
We should add swagger for the web interfaces.

**Comment by KaiVolland** [comment]
#59 adds GraphQL. If we want to remove the REST intefaces Swagger wont be needed anymore.

**Comment by marcjansen** [comment]
Have we decided on removing the REST API?

**Comment by KaiVolland** [comment]
Nope. We didn't even talked about it. But usually you go with either REST or GraphQL. I'm fine with having both. As the REST API is quite generic and finally both use sequelize to read/write stuff so we won't have problems keeping both.

**Comment by marcjansen** [comment]
I would really like to have both
